recipes-debian/libsctp/libsctp_debian.bb

```diff
@@ -0,0 +1,64 @@
+#base recipe: meta-networking/recipes-support/lksctp-tools/lksctp-tools_1.0.18.bb
+#base branch: dunfell
+#base commit: 6792ebdd966aa0fb662989529193a0940fbfee00
+
+SUMMARY = "The Linux Kernel Stream Control Transmission Protocol (lksctp) project"
+HOMEPAGE = "http://lksctp.org"
+SECTION = "net"
+LICENSE = "LGPLv2.1 & GPLv2"
```

Comment:
according to the debian copyright file, there is some BSD-3-clause license files.

```
Files: src/withsctp/*
Copyright: 2003 La Monte HP Yarroll <piggy@acm.org>
License: BSD-3-clause
```

Reply:
The log.do_compile contains the log of building `withsctp/`.
Therefore, add the BSD-3-clause license.
```
Making all in withsctp
make[3]: Entering directory '<snip>/libsctp/1.0.18-r0/lksctp-tools-1.0.18/src/withsctp'
```

Reply:
Thanks.